eng/Directory.Build.Common.props

```diff
@@ -78,6 +78,11 @@
     </NoWarn>
 
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <!-- 
+      .NET 8 now produces security warnings for nuget packages in dotnet restore. Keep these as warnings to avoid needing to update T1 library dependencies.
```

Comment:
Is there a way to scope them to warnings for T1 libraries but errors for T2 libraries?  Mostly just curious.

Reply:
Hmm yeah it seems like there would be a way to do that, let me see if I can figure it out